### PR TITLE
Add ability to hide label

### LINKED ISSUE #31: Add the ability to hide/show the progress label
Please add variable to show or hide progress label (percentage).

### PR DESCRIPTION
**Pull request by rcobelli**
Add `isLabelHidden` property. Fixes #31

### CODE REVIEW

File: Sources/CircularProgress/CircularProgress.swift [hunk 1]
```diff
@@ -8,6 +8,7 @@ public final class CircularProgress: NSView {
 	private var finishedObserver: NSKeyValueObservation?
 	private var cancelledObserver: NSKeyValueObservation?
 	private var indeterminateObserver: NSKeyValueObservation?
+	private var hiddenLabelObserver: NSKeyValueObservation?
```
Comment (sindresorhus) [inline Sources/CircularProgress/CircularProgress.swift]: This is not used anywhere.

**Comment by sindresorhus** [comment]
It still shows the label briefly when hovering over the progress circle and then out again. It also doesn't work with `.showCheckmarkAtHundredPercent = false`. Please manually test the changes thoroughly.

**Review by sindresorhus — CHANGES_REQUESTED** [review]

File: Sources/CircularProgress/CircularProgress.swift (hunk 1 above)
Comment (rcobelli) [reply Sources/CircularProgress/CircularProgress.swift]: Resolved in daf3113

**Comment by sindresorhus** [comment]
Thanks :)

**Pull request merged**